### PR TITLE
update docu for externally managed private key provisioning

### PR DESCRIPTION
**Pull request by liga-oz**
update readme

### CODE REVIEW

**Review by nenaraab — COMMENTED** [review]
Hi @liga-oz do you know this documentation?
https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config

and don't forget to adapt the `spring-security` project.

**Review by nenaraab — APPROVED** [review]
Hi @liga-oz I've incorporated my findings directly. If you are ok we can merge it to master

File: token-client/README.md [hunk 1]
```diff
@@ -168,12 +162,37 @@ To consume the `XsuaaTokenFlows` class, you simply need to `@Autowire` it like t
 private XsuaaTokenFlows xsuaaTokenFlows;
 ```
 
-For X.509 based authentication method using an externally managed certificate, `ClientCertificate` class needs to be instantiated with the external key. For `spring-xsuaa` based applications it can be easily done by overriding these values in `application.yml properties file.
-```yaml
-# For externally managed X.509 certificate
-xsuaa:
-  key: -----BEGIN RSA PRIVATE KEY-----YOUR PRIVATE KEY-----END RSA PRIVATE KEY-----
-```
+For X.509 based authentication method using an externally managed certificate, `ClientCertificate` class needs to be instantiated with the external key and `xsuaaTokenFlows` bean needs to be overwritten using this `ClientCertificate` instance. <br>Alternatively you can provide the certificate key property as command-line argument or programmatically providing default property.
```
Comment (liga-oz) [inline token-client/README.md:165]: @nenaraab then we don't mention command line argument example? do you think it's not relevant?

Comment (nenaraab) [reply token-client/README.md:165]: exactly. i see no benefit for productive usage :-)

**Pull request merged**